### ISSUE TITLE
✘ No compatible model found for 'en_core_web_trf' (spaCy v2.3.4).

### ISSUE THREAD

**Issue by P-mir** [explosion/spaCy#6459]
## How to reproduce the problem

Upgrade spacy to 2.3.4 and run the command to download a model:

```
python -m spacy download en_core_web_trf
```
results in an error:
```
✘ No compatible model found for 'en_core_web_trf' (spaCy v2.3.4).
```

## Your Environment

* Operating System: Windows
* Python Version Used: 3
* spaCy Version Used: 2.3.4

**Comment by adrianeboyd** [comment]
Hi, this model is only available for spacy v3.0+ (currently v3.0.0rc2, installed as `spacy-nightly`). For spacy v2.3.4, the available models are here: https://spacy.io/models/

(Is it possible you had installed `spacy-nightly` and then ran `pip install -U spacy` without `--pre`, which caused it to switch back to the stable release? Check that you only have one of `spacy` and `spacy-nightly` installed in your environment.)

**Comment by P-mir** [comment]
That was it thank you!
